cmd/mailslurper/config.json

```diff
@@ -1,16 +1,18 @@
 {
 	"wwwAddress": "localhost",
-	"wwwPort": 8080,
+	"wwwPort": 8081,
```

Comment:
I'm not sure this default value should change. It has been 8080 for years. And if someone needs it to be different it is fairly well documented on how to change it (I hope)